DockerFiles/README.md

```diff
@@ -0,0 +1,41 @@
+
+# How to setup the work env in docker
+
+## Install and setup stuff
+
+1. Install docker if it is not installed yet.
+2. Clone this folder.
+3. Inside this folder, build this docker file as:  
+`$ docker build --build-arg user=$USER -t work_env .`  
+It will download ros, gazebo, px4 and some utilities, so it might take time to install.
+4. ONLY NEED TO DO THIS ONCE.  
+Create a new container with the image:  
+`$ docker run --it -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix work_env bash` 
```

Comment:
Should have been `$ docker run -it -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix work_env bash`. I will fix it, no worries :)